doc/programming_guide/mic_aggregator/mic_aggregator.rst

```diff
@@ -1,3 +1,248 @@
+
+A single hardware thread contains the task and a triple buffer scheme is used to ensure there is always
+a free buffer available to write into regardless of the relative phase between the production
+and consumption of microphone samples.
+
+The `Hub` task has plenty of timing slack and is a suitable place for adding signal processing
+if needed.
+
+
+TDM Host Connection
+-------------------
+
+The TDM build supports a 16-slot TDM slave Tx peripheral from the fwk_io sub-module. In this application
+it runs at 24.576 MHz bit clock which supports 16 channels of 32 bit, 48 kHz samples per frame. 
+
+The TDM component uses a single hardware thread. 
+
+For the purpose of debugging a simple TDM 16 Master Rx component is provided. This allows the transmitted
+TDM frames from the application to be received and checked without having to connect an external
+TDM Master. It may be deleted / disconnected without affecting the core application.
+
+.. note::
+    The simple TDM 16 Master Rx component is not regression tested and is for evaluation of TDM 16 Slave Tx in this application only.
+
+
+
+USB Host Connection
+-------------------
+
+As an alternative to TDM, a USB host connection is also supported. The USB connection uses the following specifications:
+
+- USB High Speed (480 Mbps)
+- USB Audio Class 2.0
+- Asynchronous mode (audio clock is provided by the firmware)
+- 24 bit Audio slots
+- 48 kHz Sample Rate
+
+The USB host connection functionality is provided by lib_xua which is the core library of XMOS's USB Audio solution.
+
+The USB Audio subsection uses a total of four hardware threads in this application.
+
+
+Resource Usage
+==============
+
+The xcore-ai device has a total resource count of 2 x 524288 Bytes of memory and 2 x 8 hardware threads across two tiles.
+This application uses around half of the processing resources and a tiny fraction of the available memory 
+meaning there is plenty of space inside the chip for additional functionality if needed.
+
+TDM Build
+---------
+
+===== ======== =======
+Tile   Memory  Threads
+===== ======== =======
+  0    25996      5
+  1    22812      2*
+Total  48808      7 
+===== ======== =======
+
+* An additional debug TDM Master thread is used on Tile[1] by default which is not needed in a practical deployment.
+
+USB Build
+---------
+
+===== ======== =======
+Tile   Memory  Threads
+===== ======== =======
+  0    24252      4
+  1    52116      5
+Total  76368      9 
+===== ======== =======
+
+
+Board Configuration
+===================
+
+Make the following connections using flying leads:
+
+===============     ================    ==============================================
+Host Connection     Board Connection    Note
+===============     ================    ==============================================
+MIC CLK             J14 ‘00’            This is the microphone clock which is to be sent to the PDM microphones from J14.
```

Comment:
It took me a while to decipher what "J14 '00'" means. Maybe it would make more sense if I have the board in front of me? "J14 pin 0" could be clearer